src/player.h

```diff
@@ -198,14 +198,25 @@ class player : public Character, public JsonSerializer, public JsonDeserializer
         int  swim_speed() const;
         /** Maintains body wetness and handles the rate at which the player dries */
         void update_body_wetness( const w_point &weather );
-        /** Increases hunger, thirst, fatigue, stimms wearing off, dying from hunger and dying from overdose */
-        void update_needs();
-        /** Handles passive regeneration of pain and maybe hp, except sleep regeneration.
+        /** Updates all "biology" as if time between `from` and `to` passed, or by one turn */
```

Comment:
I'm gathering from the structure here that update_body() *must* be called every turn for the player, and void update_body( int from, int to ) is used to process a span of time having passed for a NPC, that needs to be documented here.  If I'm wrong, it's even important to document how it works here :)